dist/ng-formio-helper.js

```diff
@@ -751,15 +751,15 @@ angular.module('ngFormioHelper', ['formio', 'ngFormioGrid', 'ui.router'])
                 $rootScope.assignRoles = function() {
                   for (var roleName in roles) {
                     if (roles[roleName].admin) {
-                      $rootScope['is' + roles[roleName].title] = $rootScope.isAdmin = $rootScope.hasRole(roleName);
+                      $rootScope['is' + roles[roleName].title.replace(/ /g, '')] = $rootScope.isAdmin = $rootScope.hasRole(roleName);
```

Comment:
This would be better to use the ```\s``` special character which stands for "any whitespace". Can you change it to the following and we will be happy to accept.

```title.replace(/\s/g, '')```